.dockerignore

```diff
@@ -1,4 +1,7 @@
 node_modules
 npm-debug.log
 .idea
-.elasticbeanstalk
\ No newline at end of file
+.elasticbeanstalk
```

Comment:
Why was this change needed? I think this docker container needs to ignore these directories?